### ISSUE TITLE
How can I run Mask_RCNN/samples/demo.ipynb in GPU?

### ISSUE THREAD

**Issue by gkv91** [matterport/Mask_RCNN#785]
Hi,
When I ran the demo.ipynb notebook no GPU memory is utilized. What changes do I need to make to run it in GPU?

Thanks

**Comment by artisvirat** [comment]
Have you installed CUDA or any other graphical platform?

**Comment by gkv91** [comment]
yes. I have a working Titan GPU environment. Usually, we will transfer the model to GPU by model.to(device) or model.cuda(). But here I didn't find anything like that.

**Comment by artisvirat** [comment]
You do not need model.to(device), just open the jupyter notebook in your gpu virtualenv.

**Comment by gkv91** [comment]
I am doing that only. But during running, no GPU memory is utilized (checking via nvidia-smi).

**Comment by gkv91** [comment]
Linux, 
Anaconda, 
cuDNN,
no

**Comment by gkv91** [comment]
I can run other deeplearning programs without any error. So I think the environment is fine.

**Comment by artisvirat** [comment]
Did you install the gpu version of tensorflow?

**Comment by artisvirat** [comment]
Also check in the Nvidia control panel for the icon of gpu activity

**Comment by gkv91** [comment]
Ok

**Comment by gkv91** [comment]
@cardboardcode  Sorry for the late reply. My tensorflow was not gpu version. Now I can run in GPU. I have another doubt, how can I specify the GPU device, currently it is running on all GPUs available. 
Thanks

**Comment by Thoriny** [comment]
I have same problem,i use nvidia-smi to check the gpu ,but i found the gpu doesn't show work,while the program is running ,so how can i check whether i use gpu.
1.Linux2.not Anacoda 3.installed cuda &cudnn 4.yes,there is a program shows "/usr/bin/python" using gpu0 and"/home/sky/anaconda/envs/fastai/bin/python"using gpu 1,which occupy 1g ,and both 2 cards
have 16G for GPU memory. Besides tensorfow==1.40 tensorflow-gpu==1.3.0 tensorflow-tensorboard==0.1.8, I also set path for CUDA in my bashrc.

**Comment by HAMZARaouia** [comment]
if you don't have a GPU, you can run the notebook on Google Collab.
